jcommune-service/src/test/java/org/jtalks/jcommune/service/bb2htmlprocessors/UrlToLinkConvertPostProcessorTest.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (C) 2011  JTalks.org Team
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.jtalks.jcommune.service.bb2htmlprocessors;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Created by Alexey Usharovskiy on 28.12.16.
+ */
+public class UrlToLinkConvertPostProcessorTest {
+    UrlToLinkConvertPostProcessor postProcessor;
+
+    @BeforeMethod
+    public void setUp() {
+         postProcessor = new UrlToLinkConvertPostProcessor();
+    }
+
+    @Test(enabled = true)
+    void testPostProcessTextWithoutLinks() {
+        String testText = "Разнообразный и богатый опыт постоянный количественный рост и сфера нашей" +
```

Comment:
Would be great to have the text in english so that people that don't know russian don't feel awkward :)